signac/contrib/linked_view.py

```diff
@@ -39,6 +39,28 @@ def create_linked_view(project, prefix=None, job_ids=None, index=None, path=None
         if not job_ids.issubset({doc['_id'] for doc in index}):
             raise ValueError("Insufficient index for selected data space.")
 
+    key_list = [
+            k for job in jobs for k in job.statepoint().keys()
+            ]
+    value_list = [
+            v for job in jobs for v in job.statepoint().values()
+            ]
+    item_list = key_list + value_list
+    bad_chars = [os.sep, " ", "*"]
+    bad_items = [
+            item for item in item_list for char in bad_chars
+            if isinstance(item, str) and char in item
+            ]
+
+    if any(bad_items):
+        err_msg = " ".join([
+            "In order to use view, statepoints should not contain {}:".format(
+                bad_chars
+            ),
+            *bad_items
+        ])
+        raise NotImplementedError(err_msg)
```

Comment:
I would prefer this to be a `RuntimeError`.

Reply:
OK, I'll make this change!